src/main/java/io/quarkus/gizmo2/impl/MethodDescImpl.java

```diff
@@ -16,7 +16,7 @@ public sealed abstract class MethodDescImpl implements MethodDesc permits ClassM
         this.owner = owner;
         this.name = name;
         this.type = type;
-        hashCode = Objects.hash(owner, name, type);
```

Comment:
The point here is to avoid the extra allocation required by the usage of `Objects#hash()`.